.github/workflows/multiarch.yml

```
@@ -0,0 +1,19 @@
+---
+name: Multi-arch builds
```

```suggestion
name: Multi-arch Builds
```

This same nit about the UI applies to all the similar PRs. :sweat_smile: